pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -139,6 +139,22 @@ func toApiReadOnly(src bool) *ReadOnly {
 	return nil
 }
 
+// Convert_v1_Agent_To_api_Channel creates the channel for guest agent communication
+func Convert_v1_Agent_To_api_Channel(source *v1.GuestAgent, channel *Channel) error {
+
+	channel.Type = "unix"
+	channel.Source = ChannelSource{
+		Mode: "bind",
+		Path: "/var/lib/libvirt/qemu/f16x86_64.agent",
```

Comment:
what is f16x86_64?

Reply:
The value presented on the [libvirt site](https://wiki.libvirt.org/page/Qemu_guest_agent).
As far as I understand it correctly it can be basically anything valid. It serves as a unix socket path.

As for the f16 I believe it is artefact from old Fedora 16. The code works. It is only the path that is weird :).